checker/src/constant_domain.rs

```diff
@@ -637,6 +637,7 @@ pub struct ConstantValueCache {
     i128_cache: HashMap<i128, ConstantDomain>,
     u128_cache: HashMap<u128, ConstantDomain>,
     str_cache: HashMap<String, ConstantDomain>,
+    core_slice_len_function: Option<ConstantDomain>,
```

Comment:
Ok for now, but if you end up having more of these you should use some container type.

Reply:
#125